src/views/Dashboard.vue

```diff
@@ -653,6 +664,14 @@ export default {
       unassigned: 0,
       unassignedPercent: 0,
       lastMeasurement: '',
+      options: [
+        { value: 5, text: '5' },
+        { value: 10, text: '10' },
+        { value: 20, text: '20' },
+        { value: 30, text: '30', selected: true },
+        { value: 60, text: '60' },
+        { value: 90, text: '90' },
+      ],
```

Comment:
Consider adding the unit in `text`, so it reads `30 days`. 

For translations with `$t`, you can use placeholders like so: 

* https://github.com/DependencyTrack/hyades-frontend/blob/5476689620f24a158c8d9132d2fa6c954e6d4833/src/i18n/locales/en.json#L351
* https://github.com/DependencyTrack/hyades-frontend/blob/5476689620f24a158c8d9132d2fa6c954e6d4833/src/views/portfolio/tags/TaggedNotificationRuleListModal.vue#L8